addons/account/static/src/components/x2many_buttons/x2many_buttons.js

```diff
@@ -0,0 +1,61 @@
+/** @odoo-module **/
+
+import { standardFieldProps } from "@web/views/fields/standard_field_props";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { Component } from "@odoo/owl";
+
+class X2ManyButtons extends Component {
+    static template = "account.X2ManyButtons";
+    static props = standardFieldProps;
+
+    setup() {
+        this.action = useService("action");
+        this.orm = useService("orm");
+    }
+
+    async openTreeAndDiscard() {
+        const ids = this.currentField.currentIds;
+        await this.props.record.discard();
+        this.action.doAction({
+            type: "ir.actions.act_window",
+            res_model: this.currentField.resModel,
+            views: [
+                [false, "form"],
+                [false, "list"]
+            ],
+            view_mode: "list",
```

Comment:
I tested the action by clicking on the button and I arrived on a form view to create a new move.
It seems that the order here is very important and also that `view_mode` is just ignored when there is a `views` property.

Side note: don't forget to test what I am giving to you because I might be wrong/have made a mistake.
```suggestion
            views: [
                [false, "list"],
                [false, "form"],
            ],
```

Reply:
works like a charm